app-misc/hodie/Manifest

```diff
@@ -0,0 +1,2 @@
+DIST hodie-1.5.tar.gz 18337 BLAKE2B d95b94bdad332b74751939df1f3e467e9ed47d7e3603567ab8d850bb8fe75b720cf7c781576d992f22b6fe70d3693dd675fcd5e21d5d0b95611fee9bcc461fae SHA512 1262905beb1d569201fc81d1257ad6f8c1c714ad6d7a7395701dee85dd566733edc00bcd00ab85c4c161e20649d3d4ade9cd2b4252d555ac42cce1975df92a63
```

Comment:
I really think you should drop either ebuild. No point pushing two ebuilds with similar KEYWORDS and SLOTs, because users will always end up installing the later one.